app/views/tags/index.html.erb

```diff
@@ -45,7 +45,7 @@
           <div class="mt-auto">
             <button
               class="crayons-btn crayons-btn--secondary follow-action-button"
-              data-info='{"id":<%= tag.id %>,"className":"Tag", "followStyle":"secondary", "name": "<%= tag.name %>"}'>
+              data-info='<%= DataInfo.to_json(object: tag, class_name: "Tag", followStyle: "secondary") %>'>
```

Comment:
We may be able to get away with not passing "Tag" but I've seen both a Tag object and ActsAsTaggableOn::Tag object come through in my copious amounts of outside looking in debugging.  So I'm erring on the side of keeping the class_name.